api-types/lib/index.d.ts

```diff
@@ -1,19 +1,17 @@
-// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
```

Comment:
Shouldn't the linter complain since we are removing this header?

Reply:
I turned it off for this - I don't really wanna fight with having it in the built types, it should just be in our source code.